packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -147,7 +147,10 @@
 
 .actions-start,
 .actions-end {
-  @apply flex items-center;
+  @apply flex items-stretch;
+  * {
```

Comment:
I think we can skip the `*` rule because:

1. These slots are specifically for actions, which don't have children.
2. Overriding user styles for slotted content should be avoided.

Reply:
The thing is that https://github.com/Esri/calcite-design-system/blob/7ddbffb7deba19f84e815a1471186cee9164a9ff/packages/calcite-components/src/components/accordion-item/accordion-item.scss#L92 causes all items to have "items-center" so it has to be overridden which is what this line change does. How should we proceed?

Reply:
Apologies! I thought this was targeting the slotted content. 😅

Sidebar: I still think we can tidy this up to avoid having to override the universal selector elsewhere. We should consider swapping out the universal selector for explicit ones in a follow-up PR.